index.html

```diff
@@ -69,5 +77,296 @@
       </ul>
       <div class="some"></div>
     </div>
+    <div class="recent-works">
+      <div class="title">
+        <h1 class="head-title">My Recent Works</h1>
+        <ul class="line">
+          <li class="sub-line"></li>
+        </ul>
+      </div>
+      <div class="frame">
+        <div class="featured-image">
+          <div class="img"></div>
+          <div class="right-block">
+            <p class="title-post">Multi-Post Stories</p>

```

Comment:
- [ ] Kindly change this `p` tag to any of the `hx (h2-h6)` tags. It is semantic to use `hx` tags for headings. Kindly change it 👍